debug.log

```diff
@@ -0,0 +1,6 @@
+[0409/143456.650:ERROR:scoped_process_suspend.cc(31)] NtSuspendProcess: An attempt was made to access an exiting process. (0xc000010a)
```

Comment:
This file should not be in the commit, please remove it and add it to .gitignore